.travis.yml

```diff
@@ -48,12 +42,7 @@ before_install:
   - phpenv config-rm xdebug.ini
 
   - if [ $DB = 'mysql' ]; then mysql -u root -e 'CREATE DATABASE cakephp_test;'; fi
-  - if [ $DB = 'mysql' ]; then mysql -u root -e 'CREATE DATABASE cakephp_test2;'; fi
```

Comment:
The additional databases aren't needed anymore?

Reply:
Nope. I don't think we ever used them even in 3.x

Reply:
Huh, I always thought they were for cross db tests.

Reply:
They were, but I don't think we've had those in Cake 3.